### DISCUSSION TITLE
Is pdfjs-dist not supported anymore ?

### DISCUSSION THREAD

**Discussion by sayo96** [mozilla/pdf.js#18458] [Q&A] [answered]
Hey guys,
I have been trying to parse a pdf using https://github.com/mozilla/pdfjs-dist on my server. I recently noticed the archived label.
Here are my questions:
- Is pdfjs-dist no longer maintained ?
-  I would also like to know is there someway to use https://github.com/mozilla/pdf.js on node server ?

**Comment by Snuffleupagus** [comment]
***Please STOP SPAMMING unrelated issues/pull requests, and please refrain from unnecessarily pinging people.***

**Reply by GreenFlag31** [reply]
Why has it been archived?

**Comment by timvandermeij** [comment, marked as answer]
The `pdfjs-dist` repository is archived because we only distribute `pdfjs-dist` as an NPM package, we haven't pushed to the `pdfjs-dist` repository in over 1.5 years because of infrastructure issues and there wasn't a use case for it anymore that warranted further work to it. Refer to https://github.com/mozilla/pdf.js/issues/18357 for more context.

**Reply by sayo96** [reply]
Thanks for the swift response @timvandermeij . I'm looking to build a POC using pdf-dist and want to know if `pdf-dist` will still receive support even though you will only distribute it as an NPM Package.

**Reply by timvandermeij** [reply]
Yes, the `pdfjs-dist` NPM package is supported; it's purely the separate Git repository that we stopped supporting.

**Reply by sayo96** [reply]
Thank you for adding clarity
